lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -110,10 +115,12 @@ def destroy
 
     # Returns ['<map1>', '<map2>']
     def advertise_map_exist
+      fail ArgumentError if platform == :ios_xr
```

Comment:
I see why I did this now, these are mainly for the 'default' properties - where the yaml isn't _excluded. I'll rework this.